app/src/hooks/useMarkets.tsx

```diff
@@ -1,113 +1,154 @@
 import { useEffect, useState } from 'react'
 
 import { ConnectedWeb3Context } from './connectedWeb3'
-import { useContracts } from './useContracts'
-import { FEE } from '../common/constants'
-import { MarketMakerService } from '../services/market_maker'
-import { getLogger } from '../util/logger'
-import { Market, MarketWithExtraData, MarketFilters, MarketStatus } from '../util/types'
+import { useContracts, Contracts } from './useContracts'
+import { MarketWithExtraData, MarketFilters } from '../util/types'
+import { callInChunks, Range } from '../util/call_in_chunks'
 import { RemoteData } from '../util/remote_data'
 
-const logger = getLogger('Market::useMarkets')
+const EARLIEST_BLOCK_TO_CHECK = 4986777
```

Comment:
This should be an env var , we can added this value `4986777` as a default value if the env var doesn't exist.

Reply:
Done. Also added the chunk size as an env var.